js/piwik.js

```diff
@@ -3508,13 +3512,24 @@ if (typeof window.Matomo !== 'object') {
                     timings += '&pf_tfr=' + Math.round(performanceData.responseEnd - performanceData.responseStart);
                 }
 
-                if (performanceData.domInteractive && performanceData.domLoading) {
+                if (performanceData.domLoading) {
```

Comment:
In some cases `0` would be a valid value for domLoading. So might be more accurate to use something like `('domLoading' in performanceData)`.

Reply:
good point 👍